### ISSUE TITLE
For cycle 17

### ISSUE THREAD

**Issue by oscarguindzberg** [bisq-network/compensation#662]
## Summary

> _Specify the total amount of BSQ you are requesting, along with the USD total and BSQ/USD rate (don't include the brackets!):_

- **BSQ requested**: **`14925.37`** 
- USD requested: 10000 
- BSQ rate: 0.67 USD per BSQ 
- Previous compensation request (if applicable): #577  

## Contributions delivered

> _Add contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19) and roles you have performed here as new rows in the table below. Role line-items should include an asterisk (*) in the team column._

| Title | Team | USD | Link | Notes |
| --- | --- | --- | --- | --- |
| Use bisq's bitcoinj 0.15.8 | dev | 5000 | https://github.com/bisq-network/bisq/pull/4504 | |
| Extract btc to segwit addresses | dev | 5000 | https://github.com/bisq-network/bisq/pull/4509 |  |

Context: 
https://github.com/bisq-network/projects/issues/42
https://github.com/bisq-network/proposals/issues/247

## Contributions in progress

> _Provide links to work you're involved with that is still [in progress](https://github.com/bisq-network/proposals/issues/19). **This section is optional (the linter ignores it),** and is for your own benefit in keeping track of what you're doing and keeping other contributors up to date with the same._

**Comment by wiz** [comment]
This compensation proposal is subject to the terms of DAO proposal 247 https://github.com/bisq-network/proposals/issues/247 and will be paid from the special BTC multisig escrow to @oscarguindzberg as follows, pending DAO approval in Cycle 17 voting:

USD: 10000
BSQ: 10000 / .67 = 14925.37 BSQ
BTC: 14925.37 * 0.00005669 = 0.84611940, minus 0.01492537 BTC for comp req = 0.83119403 BTC net to oscar
Note: Using 30 days average 5669 sats BSQ/BTC rate from screenshot below taken at the time of compensation proposal creation

<img width="1246" alt="Screen Shot 2020-09-22 at 1 26 02" src="https://user-images.githubusercontent.com/232186/93794171-b9afd100-fc72-11ea-80e5-99508645506c.png">

**Comment by chimp1984** [comment]
What means "0.01492537 BTC for comp req" ?

**Comment by wiz** [comment]
that refers to the cost incurred by the Guarantor for coloring the BSQ since 1 BSQ requires 100 sats, which would have been incurred by Oscar if he created his own comp req like normal

**Comment by chimp1984** [comment]
Ah true. Was surprised its that much but its correct. Printing money is not cheap these days ;-)

**Comment by MwithM** [comment]
@wiz If this is pending on approving a proposal on the DAO, will this issue produce a DAO submission? Should it be tagged as "For cycle 17"? 
Let me know how to deal with this special compensation request.

**Comment by ripcurlx** [comment]
> @wiz If this is pending on approving a proposal on the DAO, will this issue produce a DAO submission? Should it be tagged as "For cycle 17"?
> Let me know how to deal with this special compensation request.

Not sure what proposal @wiz was referring to that needs to pass the DAO voting as it was approved already without putting it to the DAO for voting. @chimp1984 is reviewing @oscarguindzberg work before anything is merged into master. 
So from my side as stand in [team lead](https://bisq.wiki/Compensation#Team_Lead) I approve this request. Please submit your DAO proposal and paste the transaction ID here as a comment when complete, thanks.

**Comment by wiz** [comment]
It's not "pending" anything, it's "subject to the terms of" the linked proposal. See the proposal for the details.

**Comment by wiz** [comment]
As per bisq-network/proposals#247 the Guarantor has created DAO proposal with TXID 
`a78ec3ddd7d2b65b4a29c2429f0a0a35239516e6a7cf4b6dff15bf82ad291698` @oscarguindzberg can you please confirm this

**Comment by oscarguindzberg** [comment]
I am happy the BSQ going to the guarantor as per https://github.com/bisq-network/proposals/issues/247

**Comment by MwithM** [comment]
Closed as accepted
